src/Microsoft.OpenApi/Exceptions/OpenApiUnsupportedSpecVersionException.cs

```diff
@@ -18,7 +18,7 @@ public class OpenApiUnsupportedSpecVersionException : Exception
         /// Initializes the <see cref="OpenApiUnsupportedSpecVersionException"/> class with a specification version.
         /// </summary>
         /// <param name="specificationVersion">Version that caused this exception to be thrown.</param>
-        public OpenApiUnsupportedSpecVersionException(string specificationVersion)
+        public OpenApiUnsupportedSpecVersionException(string? specificationVersion)
```

Comment:
if we have an exception dedicated to telling users the specification version they are trying to read is not supported, having the version as the error message should be mandatory.